config/schedule.rb

```diff
@@ -13,6 +16,10 @@
   rake "order_details:auto_logout"
 end
 
+every 1.minute, roles: [:db] do
+  command "curl #{Rails.application.routes.url_helpers.admin_services_cancel_reservations_for_offline_instruments_url}"
```

Comment:
Does this have output on success? Otherwise we'll be getting cron emails every minute.